### ISSUE TITLE
GLITCHED SW NPC'S

### ISSUE THREAD

**Issue by ajberret** [dalaranwow/dalaran-wow#4613]
As it currently stands, there is no SW inkeeper or one of the AH auctioneers because a belf priest lvl 79 just mind controlled them into the SW glitch where it takes you under the map. When she released them, they went back to their original locations, only UNDER THE MAP. She would have done it for all the auctioneers if someone didnt kill her. So these NPC's will now NOT RESPAWN, as they think they are in the right place which they are, except a mile underground. 

I just came to this server, and I want you to do well. I like Algalon, and I don't want ppl to come here, see that your SW has no Inn or AH npcs, and think your server is completely glitched out. It will lose ppl, and I want this server's population base to expand, not diminish.

Please respond to this issue, and fix the glitch itself so it can't happen. This can't be tolerated. These NPC's are gone forever unless a high lvl horde gets into the glitch and kills them so they can properly respawn. And even then that belf priest could come right back and do it again.

VIDEO SHOWING THE GLITCH SPOT: (the blood elf got the npcs down there with mind control)

https://www.youtube.com/watch?v=36BuW64N2gw

[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Please see my text explanation above
2.  Please see my text explanation above
3.  Please see my text explanation above

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
DB/Creature: Gryphon Master + Auctioneer + Innkeeper - Apply Mind Controle Immunity

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4613
* commit: f11b82f128f4126ddcca5e08f748f651a4fd8c7f

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by ajberret** [comment]
Laavalambbu, one of the individuals exploiting the glitch for negative purposes.

**Comment by Rushor** [comment]
so basically we need to add a movehomeposition call once the mind control fades from the creature

**Comment by Rushor** [comment]
okay attackstart on charm remove works however CREATURE_FLAG_EXTRA_CIVILIAN prevents the movement

**Comment by Rushor** [comment]
okay i found out that it bugs for all creatures that have CREATURE_FLAG_EXTRA_CIVILIAN but without sai,

**Comment by Rushor** [comment]
Game/Unit: Add attackstart for creatures once the charm fades away + add basic SAI to all creatures with CREATURE_FLAG_EXTRA_CIVILIAN which are spawned in the world

* updates: https://github.com/dalaranwow/dalaran-wow/issues/4613
* commit: 88398faa1a75beba3b21e477de03f9ae7da14168

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Rushor** [comment]
already on PTR
